readme.md

```diff
@@ -230,6 +261,14 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+  andy = Instructor.find({name: "Andy"})
+  andy.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold"})
```

Comment:
This won't work because `.find` is asynchronous. Your `.push` should be in a callback that gets executed only after `.find` has completed. Also, `.find` will always return an array. You probably want `.findOne`